package.json

```diff
@@ -94,7 +94,7 @@
     "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
     "generate-heading-ids": "node src/scripts/generate-heading-ids.js",
     "merge-translations": "node src/scripts/merge-translations.js",
-	"check-markdown-language-tag": "node src/scripts/checkMarkdownLanguage.mjs",
+    "check-markdown-language-tag": "ts-node --esm src/scripts/checkMarkdownLanguage.mts",
```

Comment:
@pettinarip 

I am using the `ts-node` package with the `esm` flag to run the script with ES Module support

Reply:
(Reference: #6943)